Classes/Domain/Validator/Person/EmailAgainValidator.php

```diff
@@ -19,8 +18,9 @@ protected function isValid($person)
         $oldMail = $personRepository->findOneByEmail($person->getEmail());
         if ($oldMail != NULL) {
             if ($oldMail->isUnsubscribed() == 0) {
-               $langhelp = LocalizationUtility::translate('error.emailagain', 'person_manager');
-               $this->addError($langhelp, time());
+               $this->addError($this->translateErrorMessage('error.emailagain', 'person_manager'), 1620285838);
+            }else{
+                $personRepository->remove($oldMail);
```

Comment:
don't you have to persist this?

Reply:
tested it and deleted flag in db is set to 1 meaning you don't need the persistence manager